bin/e2e-test-runner

```diff
@@ -98,10 +99,16 @@ $SKIP_RECREATE_DATABASE || recreateDatabases
 $SKIP_MIGRATE || migrateDatabases
 $SKIP_SETUP_DEV || setupDev
 
-# parallel block
-# Only start webpack if not already running
-nc -vz 127.0.0.1 8234 2> /dev/null || ./bin/start-frontend &
-pnpm cypress open --config-file cypress.e2e.config.ts &
-uv pip install -r requirements.txt -r requirements-dev.txt
-python manage.py run_autoreload_celery --type=worker &
-python manage.py runserver 8080
+if [ "$START_CYPRESS" ]; then
```

Comment:
Should there be a playwright option here as well?

Reply:
well... they'll collapse once Cypress is removed

so `START_CYPRESS=0` is implicitly "i'm going to start playwright"